src/client/src/rt-components/popup/Popup.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { MouseEvent } from 'react'
+import { PopupContainer, PopupPanel, Body } from './styled'
+
+interface Props {
+  className?: string
+  open?: boolean
+  onClick?: (e: MouseEvent<HTMLDivElement>) => void
+}
+
+export default class Popup extends React.Component<Props> {
+  constructor(props: Props) {
+    super(props)
+    this.onClick = this.onClick.bind(this)
+  }
```

Comment:
Maybe use the below field syntax to remove the constructor.

Reply:
See above comment.